app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughFinalFragmentTest.kt

```diff
@@ -189,7 +189,7 @@ class WalkthroughFinalFragmentTest {
   }
 
   @Test
-  fun testWalkthroughWelcomeFragment_recyclerViewIndex1_topicSelected_yesNoButton_isDisplayedCorrectly() { // ktlint-disable max-line-length
+  fun testWalkthroughWelcomeFrag_recyclerViewIndex1_topicSelected_yesNoBtn_isDisplayedCorrectly() {
```

Comment:
we don't need `recyclerViewIndex` thing in any function name, function name just describes the action and result taken in a test.

Reply:
removing this will make some function names to be  identical and since removing the word `Fragment` from the function names makes them less than 100 chars I think we should keep `recyclerViewIndex` thing